_posts/en/newsletters/2023-02-01-newsletter.md

```diff
@@ -0,0 +1,204 @@
+
+- [Bitcoin Core #23395][] adds a `-shutdownnotify` option to `bitcoind` which
+  executes a custom user command when `bitcoind` is shut down.
+
+- [Eclair #2573][] begins accepting [keysend][topic spontaneous
+  payments] payments that don't contain a [payment secret][topic payment
+  secrets] even when Eclair advertises that the payment secret is
+  mandatory.  According to the pull request description, LND and Core
+  Lightning both send keysend payments without payment secrets.  Payment
+  secrets were designed to support [multipath payments][topic multipath
+  payments], so Eclair is leaving it up to those other node
+  implementations to ensure they only send single-path keysend payments.
+
+- [Eclair #2574][], related to the above pull request, stops including
+  payment secrets in the keysend payments it sends.  According to the
+  pull request description, LND rejects keysend payments that contain a
+  payment secret, even though such rejections are not described in
+  the [BLIP3][] specification of keysend.
+
+- [Eclair #2540][] makes several changes to how Eclair stores data about
+  funding and commitment transactions in preparation for later adding
+  support for [splicing][topic splicing].  See [#2584][eclair #2584]
+  for the current draft pull request that would add experimental
+  splicing support.
+
+- [LND #7231][] adds RPCs and options to `lncli` for signing and
+  verifying messages.  For P2PKH, the format is compatible with the
+  `signmessage` RPC first added to Bitcoin Core in 2011.  For P2WPKH and
+  P2SH-P2WPKH (also called Nested P2PKH, or NP2PKH), the same format is
+  used.  This format includes the expectation that the signature will be
+  in ECDSA format and verification requires deriving the public key from
+  the signature.  For P2TR addresses, where which would normally be used with
+  [schnorr signatures][topic schnorr signatures], it's not possible to
+  derive the public key from the signature if Bitcoin's schnorr
+  signature algorithm is used.  Instead, ECDSA signatures are generated
+  and verified for P2TR addresses.
+
+    Note: Optech generally [recommends against][p4tr new hd] using ECDSA
+    signature functions with keys intended for use with schnorr
+    signatures, but LND developers have taken [extra precautionary
+    steps][osuntokun sigs] to avoid problems.
+
+- [LDK #1878][] adds the ability to set a per-payment (rather than
+  global) `min_final_cltv_expiry` value.  This value determines the
+  maximum number of blocks the receiver has to claim a payment before it
+  expires.  The standard default value is 18 blocks but receivers can
+  request more time by setting a parameter in a [BOLT11][] invoice.
+
+    In order for LDK support this feature in combination with its unique
+    implementation of [stateless invoices][topic stateless invoices],
+    LDK encodes the value into the [payment secret][topic payment
+    secrets] that the spender is obligated to send.  It provides 12 bits
```

Comment:
(or just "required")

```suggestion
    secrets] that the spender is obliged to send.  It provides 12 bits
```

https://jakubmarian.com/obliged-vs-obligated-in-english/